<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HunterInfo</class>
 <widget class="QWidget" name="HunterInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string comment="The title of the 'Information' tab in the editor.">Information</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblHunterName">
     <property name="text">
      <string comment="Label for the hunter's name.">Hunter Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="edtHunterName">
     <property name="maxLength">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblPalicoName">
     <property name="text">
      <string comment="Label for the palico's name.">Palico Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="edtPalicoName">
     <property name="maxLength">
      <number>63</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblZenny">
     <property name="text">
      <string comment="Label for zenny.">Zenny:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spnZenny">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblResearchPoints">
     <property name="text">
      <string comment="Label for research points.">Research Points:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QSpinBox" name="spnResearchPoints">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblSteamworksFuel">
     <property name="text">
      <string comment="Label for steamworks fuel.">Steamworks Fuel:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QSpinBox" name="spnSteamworksFuel">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>99999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::AdaptiveDecimalStepType</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDoubleSpinBox" name="spnPlaytime">
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="decimals">
      <number>0</number>
     </property>
     <property name="minimum">
      <double>0.000000000000000</double>
     </property>
     <property name="maximum">
      <double>4294967295.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>3600.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QGroupBox" name="grpGuidingLands">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string comment="Label for Guiding Lands section.">Guiding Lands</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="lblLevelCapGL">
        <property name="text">
         <string comment="Label for guiding lands level cap.">Level Cap:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnLevelCapGL">
        <property name="text">
         <string comment="Button to uncap the guiding lands levels.">Uncap Guiding Lands Levels (MR100+)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblAncientForestGL">
        <property name="text">
         <string comment="Label for Ancient Forest level.">Ancient Forest:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblWildspireWasteGL">
        <property name="text">
         <string comment="Label for Wildspire Waste level.">Wildspire Waste:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblCoralHighlandsGL">
        <property name="text">
         <string comment="Label for Coral Highlands level.">Coral Highlands:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblRottenVeilGL">
        <property name="text">
         <string comment="Label for Rotten Veil level.">Rotten Veil:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblEldersRecessGL">
        <property name="text">
         <string comment="Label for Elder's Recess level.">Elder's Recess:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblHoarfrostReachGL">
        <property name="text">
         <string comment="Label for Hoarfrost Reach level.">Hoarfrost Reach:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spnAncientForestGL">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spnWildspireWasteGL">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spnCoralHighlandsGL">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spnRottenVeilGL">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="spnEldersRecessGL">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spnHoarfrostReachGL">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QLabel" name="lblNoteGL">
        <property name="text">
         <string comment="Note on the caps for each region level.">NOTE: The level caps for each region are the highest you've previously unlocked.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblPlaytime">
     <property name="text">
      <string comment="Label for hunter playtime.">Playtime:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QSpinBox" name="spnHunterHR">
     <property name="maximum">
      <number>99999999</number>
     </property>
     <property name="singleStep">
      <number>1000</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::DefaultStepType</enum>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QSpinBox" name="spnHunterMR">
     <property name="maximum">
      <number>99999999</number>
     </property>
     <property name="singleStep">
      <number>2000</number>
     </property>
     <property name="stepType">
      <enum>QAbstractSpinBox::DefaultStepType</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblHunterHR">
     <property name="text">
      <string comment="Label for hunter HR.">HR:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblHunterMR">
     <property name="text">
      <string comment="Label for hunter MR.">MR:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>edtHunterName</tabstop>
  <tabstop>edtPalicoName</tabstop>
  <tabstop>spnZenny</tabstop>
  <tabstop>spnResearchPoints</tabstop>
  <tabstop>spnSteamworksFuel</tabstop>
  <tabstop>spnPlaytime</tabstop>
  <tabstop>spnHunterHR</tabstop>
  <tabstop>spnHunterMR</tabstop>
  <tabstop>btnLevelCapGL</tabstop>
  <tabstop>spnAncientForestGL</tabstop>
  <tabstop>spnWildspireWasteGL</tabstop>
  <tabstop>spnCoralHighlandsGL</tabstop>
  <tabstop>spnRottenVeilGL</tabstop>
  <tabstop>spnEldersRecessGL</tabstop>
  <tabstop>spnHoarfrostReachGL</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>edtHunterName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>HunterInfo</receiver>
   <slot>HunterNameChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edtPalicoName</sender>
   <signal>textEdited(QString)</signal>
   <receiver>HunterInfo</receiver>
   <slot>PalicoNameChange(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnZenny</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>ZennyChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnResearchPoints</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>ResearchPointsChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnLevelCapGL</sender>
   <signal>clicked()</signal>
   <receiver>HunterInfo</receiver>
   <slot>UncapGuidingLands()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>245</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnSteamworksFuel</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>SteamworksFuelChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnAncientForestGL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>RegionalLevelGLChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnWildspireWasteGL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>RegionalLevelGLChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>235</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnCoralHighlandsGL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>RegionalLevelGLChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnRottenVeilGL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>RegionalLevelGLChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>286</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnEldersRecessGL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>RegionalLevelGLChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>311</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnHoarfrostReachGL</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>RegionalLevelGLChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>246</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnPlaytime</sender>
   <signal>valueChanged(double)</signal>
   <receiver>HunterInfo</receiver>
   <slot>PlaytimeChange(double)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnHunterHR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>HRChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spnHunterMR</sender>
   <signal>valueChanged(int)</signal>
   <receiver>HunterInfo</receiver>
   <slot>MRChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>229</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>HunterNameChange(QString)</slot>
  <slot>PalicoNameChange(QString)</slot>
  <slot>ZennyChange(int)</slot>
  <slot>ResearchPointsChange(int)</slot>
  <slot>UncapGuidingLands()</slot>
  <slot>SteamworksFuelChange(int)</slot>
  <slot>RegionalLevelGLChange(int)</slot>
  <slot>PlaytimeChange(double)</slot>
  <slot>HRChange(int)</slot>
  <slot>MRChange(int)</slot>
 </slots>
</ui>
